docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
```

Comment:
A description could be useful here. They are shown in embeds and search results. Not a must, but I'd recommend it.

Reply:
None of the other pages seem to have a description.
How would you like me to add one?

Reply:
I'm trying to add descriptions to more pages, besides the few I've already done. Something descriptive of the general topic is fine; I suppose mentioning the specific printer could be important as the previews don't actually include the grandparent name

Reply:
Looks good! Cheers.